_posts/2019-06-03-deploy-docker-to-gcp.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Deploying a Dockerized App on GCP
+logoUrl: /assets/img/guides/docker_image.png
+excerpt: Before we can deploy a dockerized app, we need to first create one. For the purposes of this guide we will create a basic Node.js app that responds to requests on port 8080.
+---
```

Comment:
Ah, interesting... So you're trying to use Jekyll's native support for Markdown and blog posts here. That's a clever solution!

Note that we also want to be able to use AsciiDoc as per https://github.com/gruntwork-io/mission-control/blob/master/spacecenter/rfc/launchpad-docs-guides-catalog-design.md. Perhaps one of the Jekyll AsciiDoc plugins can help (e.g., [this one](https://github.com/gruntwork-io/module-vpc/blob/deployment-guide/production-deployment-guide.asciidoc))?

See https://github.com/gruntwork-io/module-vpc/pull/73 for a draft of a full production deployment guide written with AsciiDoc (albeit, not yet using most of AsciiDoc's features).

Reply:
Okay cool. Using the Jekyll AsciiDoc plugin will cater for AsciiDoc